src/test/java/net/datafaker/integration/FakerIntegrationTest.java

```diff
@@ -170,6 +170,7 @@ void testAllFakerMethodsThatReturnStrings(Locale locale, Random random) throws E
         testAllMethodsThatReturnStringsActuallyReturnStrings(faker.kaamelott());
         testAllMethodsThatReturnStringsActuallyReturnStrings(faker.leagueOfLegends());
         testAllMethodsThatReturnStringsActuallyReturnStrings(faker.lebowski());
+        testAllMethodsThatReturnStringsActuallyReturnStrings(faker.localeFaker());
```

Comment:
There is a contract for `java.util.Locale#getDisplayName()` saying that

> depending on which fields are specified in the locale. The field separator in the above parentheses, denoted as a comma character, may be localized depending on the locale. If the language, script, country, and variant fields are all empty, this function returns the empty string.

that's the reason of failing. We cannot  say that display name is always not empty